pkg/apiserver/statement/service.go

```diff
@@ -89,23 +97,30 @@ func (s *Service) configHandler(c *gin.Context) {
 }
 
 // @Summary Update statement configurations
-// @Param request body statement.Config true "Request body"
+// @Param request body statement.EditableConfig true "Request body"
 // @Success 204 {object} string
 // @Router /statements/config [post]
 // @Security JwtAuth
 // @Failure 401 {object} utils.APIError "Unauthorized failure"
 func (s *Service) modifyConfigHandler(c *gin.Context) {
-	var req Config
-	if err := c.ShouldBindJSON(&req); err != nil {
+	var err error
+	var config EditableConfig
+	if err = c.ShouldBindJSON(&config); err != nil {
 		utils.MakeInvalidRequestErrorFromError(c, err)
 		return
 	}
 	db := utils.GetTiDBConnection(c)
-	err := updateStmtConfig(db, &req)
+
+	if !config.Enable {
+		err = db.Exec(buildConfigUpdateSQL(&config, "Enable")).Error
```

Comment:
What does this Enable do?

Reply:
Extract fields from struct, only the extract fields will be left in SQL.

Like this:
```
func (t *testConfigSuite) Test_buildConfigUpdateSQL_extract_fields(c *C) {
	testConfigStmt := "SET @@GLOBAL.tidb_enable_stmt_summary = true"
	c.Assert(buildConfigUpdateSQL(&testConfig{Enable: true, RefreshInterval: 1800}, "Enable"), Equals, testConfigStmt)
}
```